src/modules/users/components/UserTokenActivationButton/UserTokenActivationDisplay.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { BigNumber } from 'ethers/utils';
+
+import { getFormattedTokenValue } from '~utils/tokens';
+import Numeral from '~core/Numeral';
+
+import styles from './UserTokenActivationButton.css';
+import { UserToken } from '~data/generated';
+
+const displayName =
+  'users.UserTokenActivationButton.UserTokenActivationDisplay';
+
+interface Props {
+  nativeToken?: UserToken;
+  inactiveBalance: BigNumber;
+  totalBalance: BigNumber;
+}
+
+const UserTokenActivationDisplay = ({
+  nativeToken,
+  inactiveBalance,
+  totalBalance,
+}: Props) => {
+  const formattedTotalBalance = getFormattedTokenValue(
+    totalBalance,
+    nativeToken?.decimals,
+  );
+
+  return (
+    <div>
+      <span
+        className={`${styles.dot} ${
+          (inactiveBalance.gt(0) || totalBalance.isZero()) && styles.dotInactive
```

Comment:
We have a library to conditionally add styles like you are doing here but in a more readable way, it's called `classnames`. You can search for examples in the dapp by looking for `import classnames` :)